### ISSUE TITLE
Invalid Doubles

### ISSUE THREAD

**Issue by Messydesk** [dmstocking/couch-tatertot#1]
Hey there,
Installed couchtatertot from Play store today on a new device, and get the following error:
Error Retrieving Results ERROR: java.lang.NumberFormatException: Invalid double: "08c111f36fe76be37dd989abd5a0f7e5"

Any ideas?

**Comment by dmstocking** [comment]
Your having this problem too? GAR! Can you save the output from http://your-server:your-port/api/your-api-key/movie.list/ and send it to me? I can't even find a place were I use a double, so why does this error appear.

**Comment by Messydesk** [comment]
Sorry man, only slightly technically inclined... do you want me to visit that address on the device or the system CouchPotato runs on?

**Comment by dmstocking** [comment]
You can just do it via Firefox/Chrome/IE/whatever you use (I suggest a desktop). It should just be a giant wall of text.

Something like {"movies": [{"profile": {"core": false, "hide": null, "order": -2, "ty ............................... so on and so forth.

**Comment by dmstocking** [comment]
I just noticed that my old comment got changed by the git markdown. I changed it so that you can actually read it now.

**Comment by Messydesk** [comment]
Prefect... Got the results here

EDIT FROM BUTTINK: I got it. I also removed it from this post for your privacy.

**Comment by dmstocking** [comment]
I found the problem. One of the fields for release info is "id". I thought this was an internal couchpotato id, but its an id from the website the release is from. The fix should be out in like 10 minutes (although Google Play takes like 2 hours to actually send it out).

**Comment by Messydesk** [comment]
Beauty.. I look froward to it... It's a great app, I was running the Github install and then with the new device bought and rated through the Play store.. happy to pay for good stuff!

**Comment by dmstocking** [comment]
Fixed by https://github.com/Buttink/couch-tatertot/commit/367d02085df3cffdb52aac4bfb8678c1ff6cac29

**Comment by dmstocking** [comment]
Opening again until I confirm this is closed. BTW, there was another problem with size which was also fixed.

**Comment by Messydesk** [comment]
Hey Buttink, I've reinstalled from github 1.2.2 and I still get the same invalid double message... this time with invalid double:"47b3e1e93d03e784038c92b43e1651e8"

**Comment by dmstocking** [comment]
Do the same thing you did last time (get the movie.list), except send it to me as a file attachment in a email (Buttink@gmail.com) instead of inside this thread. I will take a look at it. This is really annoying. I really don't want users to run into these problems that I cannot reproduce >.>

**Comment by Messydesk** [comment]
okay... not sure what I did differently, I uninstalled the most recent github version, went back to Google Play and re-installed. put my settings in again, copying from the same settings document for the api key etc... that I had used last time... and Voila.  This is on a Nexus 7, I haven't had any problems with an HTC Desire HD running jellytime that I am using as well. I'll close this issue, but if I run into the problem again I'll let you know.
